### PR TITLE
modified clustermanagementaddon

### PR DESCRIPTION
**Pull request by jc-berger**
See Jira: https://issues.redhat.com/browse/ACM-12261

### CODE REVIEW

File: business_continuity/backup_restore/backup_arch.adoc [hunk 1]
```diff
@@ -22,7 +22,9 @@ View the following ordered list of the cluster backup and restore process for id
...
 
-- Exclude all of the following resources that are a part of the included API groups, but are either not needed or are being recreated by owner resources that are also backed up: `clustermanagementaddon.addon.open-cluster-management.io` ,`backupschedule.cluster.open-cluster-management.io`, `restore.cluster.open-cluster-management.io`, `clusterclaim.cluster.open-cluster-management.io`, `discoveredcluster.discovery.open-cluster-management.io`.
+- Exclude all of the following resources that are a part of the included API groups, but are either not needed or are being recreated by owner resources that are also backed up: `backupschedule.cluster.open-cluster-management.io`, `restore.cluster.open-cluster-management.io`, `clusterclaim.cluster.open-cluster-management.io`, `discoveredcluster.discovery.open-cluster-management.io`.
```
Comment (swopebe) [inline business_continuity/backup_restore/backup_arch.adoc]: This line is really confusing. 

1st, this should be a list (consider that for tech debt if we cannot fix it now, that should not be entered like this but I recognize you didn't create that)
2nd, the description of this list is not clear right off the bat and very wordy.

File: business_continuity/backup_restore/backup_arch.adoc [hunk 2]
```diff
@@ -22,7 +22,9 @@ View the following ordered list of the cluster backup and restore process for id
...
+
+- *Note:* If you modify `ClusterManagementAddon`, add the label, `cluster.open-cluster-management.io/backup` to trigger the backup.
```
Comment (swopebe) [inline business_continuity/backup_restore/backup_arch.adoc]: please remove _trigger_ and turn around this sentence for clarity:

**If you modify `ClusterManagementAddon`, add the `cluster.open-cluster-management.io/backup` label to start the backup.** (or similar)

Anytime there is a `name, comma situation`, see if you can make it more concise and clearer.

**Review by swopebe — CHANGES_REQUESTED** [review]
Comment for possible tech debt if not now, comment for concise language and asking for clearer description.

File: business_continuity/backup_restore/backup_arch.adoc [hunk 3]
```diff
@@ -12,17 +12,33 @@ The cluster backup and restore operator solution provides backup and restore sup
...
 
-- Exclude all resources from the following API groups: `internal.open-cluster-management.io`, `operator.open-cluster-management.io`, `work.open-cluster-management.io`, `search.open-cluster-management.io`, `admission.hive.openshift.io`, `proxy.open-cluster-management.io`, `action.open-cluster-management.io`, `view.open-cluster-management.io`, `clusterview.open-cluster-management.io`, `velero.io`.
-
-- Exclude all of the following resources that are a part of the included API groups, but are either not needed or are being recreated by owner resources that are also backed up: `clustermanagementaddon.addon.open-cluster-management.io` ,`backupschedule.cluster.open-cluster-management.io`, `restore.cluster.open-cluster-management.io`, `clusterclaim.cluster.open-cluster-management.io`, `discoveredcluster.discovery.open-cluster-management.io`.
+- Exclude all resources from the following API groups:
```
Comment (swopebe) [inline business_continuity/backup_restore/backup_arch.adoc:23]: This is so much easier to digest.

**Review by swopebe — APPROVED** [review]
the list is a game changer. TY

File: business_continuity/backup_restore/backup_arch.adoc [hunk 4]
```diff
@@ -12,17 +12,33 @@ The cluster backup and restore operator solution provides backup and restore sup
...
+** `clusterview.open-cluster-management.io`
+** `velero.io`
+
+- Exclude all of the following resources that are a part of the included API groups, but are either not needed or are being recreated by owner resources that are also backed up: 
+** `backupschedule.cluster.open-cluster-management.io`
+** `restore.cluster.open-cluster-management.io`
+** `clusterclaim.cluster.open-cluster-management.io`
+** `discoveredcluster.discovery.open-cluster-management.io`
+
+- *Note:* If you change `ClusterManagementAddon`, add the label, `cluster.open-cluster-management.io/backup` to initiate the backup.
```
Comment (swopebe) [inline business_continuity/backup_restore/backup_arch.adoc:41]:
```suggestion
- *Note:* If you change `ClusterManagementAddon`, add the `cluster.open-cluster-management.io/backup` label to initiate the backup. 
```

File: business_continuity/backup_restore/backup_arch.adoc [hunk 5]
```diff
@@ -12,17 +12,33 @@ The cluster backup and restore operator solution provides backup and restore sup
...
-
-- Exclude all of the following resources that are a part of the included API groups, but are either not needed or are being recreated by owner resources that are also backed up: `clustermanagementaddon.addon.open-cluster-management.io` ,`backupschedule.cluster.open-cluster-management.io`, `restore.cluster.open-cluster-management.io`, `clusterclaim.cluster.open-cluster-management.io`, `discoveredcluster.discovery.open-cluster-management.io`.
+- Exclude all resources from the following API groups: 
+** `internal.open-cluster-management.io`
+** `operator.open-cluster-management.io`
+** `work.open-cluster-management.io`
+** `search.open-cluster-management.io`
+** `admission.hive.openshift.io`
+** `proxy.open-cluster-management.io`
+**  `action.open-cluster-management.io`
```
Comment (zhujian7) [inline business_continuity/backup_restore/backup_arch.adoc:30]:
```suggestion
** `action.open-cluster-management.io`
```

**Review by birsanv — CHANGES_REQUESTED** [review]
sorry for the late reply but I just saw this change request

The changes are not needed; any doc update for the addon backup change has been covered here https://issues.redhat.com/browse/ACM-12297
PR https://github.com/stolostron/rhacm-docs/pull/6573

There are no other changes needed for the main hub backup doc

@zhujian7 any doc update left to be done is for the hypershift backup doc, which I think is addressed by @rokej with another PR

**Comment by jc-berger** [comment]
Closing this PR as these efforts are unneeded for Business Continuity. We can use the Jira to discuss similar changes but to the Hypershift Docs. Thank you all for your patience :)

**Pull request closed without merging**